src/Aspire.ProjectTemplates/templates/aspire-apphost/.template.config/template.json

```diff
@@ -47,93 +47,137 @@
       "type": "bind",
       "binding": "HostIdentifier"
     },
-    "appHttpPort": {
+    "appHostHttpPort": {
       "type": "parameter",
       "datatype": "integer",
-      "description": "Port number to use for the HTTP endpoint in launchSettings.json of the App project."
+      "description": "Port number to use for the HTTP endpoint in launchSettings.json of the AppHost project."
     },
-    "appHttpPortGenerated": {
+    "appHostHttpPortGenerated": {
       "type": "generated",
       "generator": "port",
       "parameters": {
         "low": 15000,
```

Comment:
The logic for the port ranges here is:
- HTTP:
  - AppHost dashboard HTTP port: 15000-15300
      *This aligns with the default HTTP port range for ASP.NET Core project templates of 5000-5300*
  - AppHost OTLP HTTP port: 19000-19300
  - AppHost resource service HTTP port: 20000-20300
- HTTPS:
  - AppHost dashboard HTTPS port: 17000-17300
      *This aligns with the default HTTPS port range for ASP.NET Core project templates of 7000-7300*
  - AppHost OTLP HTTPS port: 21000-21300
  - AppHost resource service HTTPS port: 22000-22300

Reply:
Should apphost template default for NoHttps also be changed to false?

Reply:
Thanks, fixed!